wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx

```diff
@@ -179,31 +179,34 @@ const TokenList = (props: Props) => {
       filterFn={(token, query) => {
         if (query.length === 0) return true;
 
+        const chain = props.selectedChainConfig.key;
+
         // Exclude frankenstein tokens with no balance
         const balance = Number(balances?.[token.key]?.balance);
-        if (
-          isFrankensteinToken(token, props.selectedChainConfig.key) &&
-          !balance
-        ) {
+        if (isFrankensteinToken(token, chain) && !balance) {
           return false;
         }
 
         // Exclude wormhole-wrapped tokens with no balance
-        if (
-          props.isSource &&
-          isWrappedToken(token, props.selectedChainConfig.key) &&
-          !balance
-        ) {
+        if (props.isSource && isWrappedToken(token, chain) && !balance) {
           return false;
         }
 
         const queryLC = query.toLowerCase();
-        return Boolean(
-          token.symbol?.toLowerCase().includes(queryLC) ||
-            getDisplayName(token, props.selectedChainConfig.key)
-              .toLowerCase()
-              .includes(queryLC),
-        );
+
+        const symbolMatch = token.symbol?.toLowerCase().includes(queryLC);
```

Comment:
Nitpick: If there is a symbolMatch, you can return here without checking other matches. Same as displayNameMatch below. However, if you think having one return for all cases makes the code more organized and readable, I'm all for that too 👍